### PR TITLE
Information about patching 3rd parties

### PR DESCRIPTION
**Pull request by vkjr**
### Summary

Docs updated with information on how we use Nix for patching 3rd parties instead of doing it nodejs-way

status: ready

### CODE REVIEW

**Review by siddarthkay — APPROVED** [review]

**Review by rahulpsingh — APPROVED** [review]

**Review by flexsurfer — CHANGES_REQUESTED** [review]
we need to discuss this, could you give an example ?

**Comment by vkjr** [comment]
> we need to discuss this, could you give an example ?

@flexsurfer, elaborated with more precise wording and examples. Wdyt?

File: doc/patching-3rdparties.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
 
-If 3rd party dependency has an issue and fix is not yet released (or we can't switch to a new release), local patch can be applied. 
-Status doesn't used nodejs-way of applying patches. Instead we use Nix.
+## Libraries
+If 3rd party library has an issue and fix is not yet released (or we can't switch to a new release), we use forks. Fix should be commited to the fork and tagged.
+
+Example: [`react-native-hole-view`](https://github.com/status-im/react-native-hole-view#refs/tags/v2.1.1-status)
+
+## React Native
+When patch need to be applied to React Native itself Status does patching with Nix instead of doing it nodejs-way.
```
Comment (flexsurfer) [inline doc/patching-3rdparties.md]: what is nodejs-way ?

Comment (siddarthkay) [reply doc/patching-3rdparties.md]: https://www.npmjs.com/package/patch-package

Comment (siddarthkay) [reply doc/patching-3rdparties.md]: > what is nodejs-way ?
https://www.npmjs.com/package/patch-package

Comment (vkjr) [reply doc/patching-3rdparties.md]: Things like this - https://www.npmjs.com/package/patch-package

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**